rotkehlchen/history/price.py

```diff
@@ -103,17 +105,20 @@ def __new__(
             data_directory: Path = None,
             cryptocompare: 'Cryptocompare' = None,
             coingecko: 'Coingecko' = None,
+            manual_price_oracle: ManualPriceOracle = None,
```

Comment:
Why do you need this given as input? Will we have multiple places where this is used like we do with cryptocompare and coingecko? I don't think so. Why not just initialize the `ManualPriceOracle` inside the `PriceHistorian()` constructor?